x-pack/plugins/cloud_security_posture/common/runtime_mappings/get_package_policy_id_mapping.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MappingRuntimeFields } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+
+export const getPackagePolicyIdRuntimeMapping = (): MappingRuntimeFields => ({
+  package_policy_identifier: {
+    type: 'keyword',
+    script: {
+      source: `
+        def packagePolicyIdAvailable = doc.containsKey("cloud_security_posture.package_policy.id") &&
```

Comment:
We will return null if the package policy id is missing.

Reply:
would be better to explicitly return the value. that way we don't need to assume how painless scripts works. don't you agree?

Reply:
I tried to return `null` and the query failed. I do have a check in case there are no buckets for package_policy_id  `package_policy_id: account.package_policy_id?.buckets[0]?.key ?? null,`